presto-docs/src/main/sphinx/release/release-334.rst

```diff
@@ -0,0 +1,87 @@
+* Fix dynamic filter failures in complex spatial join queries. (:issue:`3694`)
+* Add :doc:`/connector/oracle`. (:issue:`1959`)
+* Add :doc:`/connector/pinot`. (:issue:`2028`)
+* Add support for modern HTTP forwarded headers. Processing of HTTP forwarded headers are now controlled by the
+  ``http-server.process-forwarded`` configuration, and the old ``http-server.authentication.allow-forwarded-https`` and
+  ``dispatcher.forwarded-header`` configurations are no longer supported. (:issue:`3714`)
+
+CLI Changes
+-----------
+
+* Improve rendering of elapsed time for short queries. (:issue:`3311`)
+
+Web UI Changes
+--------------
+
+* Add fixed, certificate, JWT and Kerberos to UI authentication. (:issue:`3433`)
+* Show join distribution type in Live Plan. (:issue:`1323`)
+
+JDBC Driver Changes
+-------------------
+
+* Improve performance of ``DatabaseMetaData#getColumns`` method in case when parameters
+  contain unescaped ``%`` or ``_``. (:issue:`1620`)
+
+Elasticsearch Connector Changes
+-------------------------------
+
+* Fix failure when executing ``SHOW CREATE TABLE``. (:issue:`3718`)
+* Improve performance for ``count(*)`` queries. (:issue:`3512`)
+* Add support for raw Elasticsearch queries. (:issue:`3735`)
+
+Hive Connector Changes
+----------------------
+
+* Fix creation of external tables using ``CREATE TABLE AS``. Previously such table was created
+  as managed and with default location. (:issue:`3755`)
+* Fix incorrect table statistics for new external table. (:issue:`3819`)
+* Prevent Presto from starting when cache fails to initialize. (:issue:`3749`)
+* Fix race condition that could cause caching to be unwillingly disabled. (:issue:`3729`, :issue:`3810`)
+* Fix malformed reads when caching parallel warmup is enabled. (:issue:`3772`)
+* Fix evicting of cached data while still under size eviction threshold. (:issue:`3772`)
+* Improve performance when creating unpartitioned external tables over large data sets. (:issue:`3624`)
+* Leverage Parquet file statistics when reading decimal columns. (:issue:`3581`)
+* Change type of ``$file_modified_time`` hidden column from ``bigint`` to ``timestamp with timezone type``. (:issue:`3611`)
+* Add caching support for HDFS and Azure. (:issue:`3772`)
+* Fix S3 connection pool depletion when caching parallel warmup is enabled. (:issue:`3772`)
+* Don't initialize cache on coordinator by default. (:issue:`3820`)
+* Use caching asynchronous read mode by default. (:issue:`3799`)
+* Add session property for disabling Hive cache. (:issue:`3843`)
+* Allow caching delegation token for Hive thrift metastore. This can be configured with
+  the ``hive.metastore.thrift.delegation-token.cache-ttl`` and ``hive.metastore.thrift.delegation-token.cache-maximum-size``
+  configuration properties. (:issue:`3771`)
+
+MemSQL Connector Changes
+------------------------
+
+* Include :doc:`/connector/memsql` in the server build. (:issue:`3743`)
```

Comment:
```suggestion
* Include :doc:`/connector/memsql` in the server build.
  Previously the connector was not part of the project's release artifacts. (:issue:`3743`)
```

Reply:
"release artifacts" is not correct. I think we can just remove the second sentence.

Reply:
How about:

> Include ... in the server packaging.

or

> Include ... in the server tarball and RPM

Reply:
The second one looks good